app/decorators/question_decorator.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true.
+
+class QuestionDecorator < ApplicationDecorator
```

Comment:
I didn't catch this earlier but this should be namespaced to `Odk` since it's only related to ODK stuff. You can/should then also use the ODK::DecoratorFactory thing to instantiate.

Reply:
thanks